frontend/src/metabase-lib/parameters/utils/parameter-values.js

```diff
@@ -42,7 +44,12 @@ export function normalizeParameter(parameter) {
 export function normalizeParameters(parameters) {
   return parameters
     .filter(parameter => _.has(parameter, "value"))
-    .map(normalizeParameter);
+    .map(({ type, value, target, id }) => ({
```

Comment:
Reverts a change made by the previous PR. Unfortunately the backend treats parameter lists and single parameter differently, and passing `name` and `slug` here will break some cases with question parameters.